.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: node_js
 sudo: false
 node_js:
-  - "6.11.3"
+  - 4
```

Comment:
Node 4 EOL today, otherwise looks good

Reply:
@damienomurchu I think until it is valid and supported in RHMAP we need still checking it.

Reply:
@camilamacedo86 +1 - we should probably leave it in as long as its supported in the studio